MAINTAINERS.rst

```diff
@@ -19,14 +25,8 @@ Pre-release test plan
    - Fedora
 
 3. Build alpha N (N=1 to start, then N=2 if you need more commits, etc.) and
-   upload to pypi.
-
-   Make sure to check if west.manifest.SCHEMA_VERSION also needs an update
-   before uploading. ::
-
-     git clean -ffdx
-     WEST_VERSION=X.YaN python3 setup.py sdist bdist_wheel
-     twine upload -u zephyr-project dist/*
```

Comment:
Why are the example commands removed ?

Reply:
>  Why are the example commands removed ?

They weren't. There are two copies. I eliminated the redundant copy, leaving the single point of truth in the details section.